### DISCUSSION TITLE
How can I get Multiple source in videojs player using JavaScript

### DISCUSSION THREAD

**Discussion by sajedulsakib001** [videojs/video.js#7409] [Q&A] [answered]
I am trying to add multiple video source using this JavaScript file:
[https://github.com/sajedulsakib001/VideoJs/blob/main/script.js](https://github.com/sajedulsakib001/VideoJs/blob/main/script.js) 
But I'm getting an error (No compatible source was found for this media.), How can i fix it?

**Comment by mister-ben** [comment, marked as answer]
You need an array of objects rather than a JSON string

**Reply by sajedulsakib001** [reply]
Thanks for your reply. I have changed it to array. It's working!!!
